stdlib/private/StdlibUnittest/InterceptTraps.cpp

```diff
@@ -26,6 +26,9 @@
 
 #include "swift/Runtime/Config.h"
 
+// no signals support on WASI yet, see https://github.com/WebAssembly/WASI/issues/166
+#if !defined(__wasi__)
```

Comment:
I think that it's better to preprocess away the entire file.  I don't remember there being any symbols referenced from here.